lib/utils/validate-addons.js

```diff
@@ -1,14 +1,27 @@
 const exists = require('./npm-exists');
 const resolvePackages = require('./resolve-packages');
 
-module.exports = (addons) => addons.map( pkg => checkExistence(pkg));
+
+/*
+* @function validateAddons
+*
+* Loops through an array and checks if a package is registered
+* on npm and throws an error if it is not from @checkExistence
+*
+* @param { Array } pkg - Array of packages to check existence of
+* @returns { <Function|Error> } resolvePackages - Returns an process to install the pkg
+*/
+
+module.exports = function validateAddons(addons) {
+	return addons.map( pkg => checkExistence(pkg));
+};
 
 /*
 * @function checkExistence
```

Comment:
I would export this function and do the `map` elsewhere. Should we rename it to `npmPackageExist`?